### ISSUE TITLE
Flatt grassblock planes generates in Deep Ocean / Ocean areas

### ISSUE THREAD

**Issue by wimplaw** [orangeadam3/terra121#4]
You probably already know this based on what it says in the known problems list on curse. 
But just in case it may be of help:

I teleported to norway and to my home town:
https://www.openstreetmap.org/#map=13/61.5930/5.0212&layers=N

The norwegian sea and fjords spawns as flatt grassland at Y=0. 
There is generating some water following the coast line but only a few blocks wide.

I tried teleporting further out from the coast to see if there was any water generation, and after a few kilometres there seem to generate some deep waters.

Also the biome generated in norway is Swamp this should problably be taiga or something similar.
But as you mentioned in the description, the biomes are somtimes wrong at this point in time so this may be redundant information. :)

**Comment by orangeadam3** [comment]
Yeah, the water is just based on elevation which is not a good permanent solution. The updateaquatic branch fixes a lot of this and adds inland rivers and lakes but it has its own problems. I can look into the soil type and climate around your town to fix the biome classifier. My guess is that your town has the soil to be a swamp except it's too cold. We had a similar problem with deserts in Canada earlier.

**Comment by wimplaw** [comment]
I checked some sea maps of the area at https://www.norgeskart.no/?&_ga=2.186632617.164698678.1583278525-857528959.1583278525#!?zoom=13&lon=-28077.25&lat=6867964.02&project=dekning&layers=1008

The elevation goes down to 400 meters bellow sea level on the south side of that town, and the sea further out west seem to be mostly 300 meters bellow sea level 
I also checked out the sognefjord wich is over 1000 meters deep and seems to be generating just as flat.

Considering its based on elevation I think it is somthing wierd going on thats cousing the sea to generate like that? 
unless there is somthing wrong with the elevation data?

**Comment by orangeadam3** [comment]
It is probably the terrain dataset, it has lots of issues because the ocean depth data is awkwardly merged with the more precise land area which causes issues especially around shorelines. The new water updates should at least put the ocean in the right place.
